base/REPL.jl

```diff
@@ -958,7 +956,7 @@ type StreamREPL <: AbstractREPL
     waserror::Bool
     StreamREPL(stream,pc,ic,ac) = new(stream,pc,ic,ac,false)
 end
-StreamREPL(stream::IO) = StreamREPL(stream, julia_green, Base.input_color(), Base.answer_color())
+StreamREPL(stream::IO) = StreamREPL(stream, Base.text_color[:green], Base.input_color(), Base.answer_color())
```

Comment:
Oh, if the boldness is being applied then it doesn't really matter. I remember now that I had to add that here: https://github.com/JuliaLang/julia/pull/18628/files#diff-8dc8200a114faecb709b7b96e96fc0cfR632.